code/hud/hudshield.cpp

```diff
@@ -590,14 +590,16 @@ void HudGaugeShield::showShields(object *objp, int mode)
 
 		vm_angles_2_matrix(&object_orient, &rot_angles);
 
-		gr_screen.clip_width = 112;
-		gr_screen.clip_height = 93;
+		static const int cw = 112;
```

Comment:
This should only be `const` and the variable name should be more descriptive and upper case since it's a constant.